src/order/order.service.spec.ts

```
@@ -0,0 +1,297 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { OrderService } from './order.service';
+import { PrismaService } from '../prisma/prisma.service';
+import { mapPrismaOrderToDomain, calculateTotalAmount as actualCalculateTotalAmount, createCancelledOrderStatus as actualCreateCancelledOrderStatus } from './order.mapper';
+import { CreateOrderDto, UpdateOrderStatusDto, OrderItemDto } from './dto/order.dto';
+import { OrderId, OrderStatus, OrderStatusEnum, TotalAmount, OrderItem as DomainOrderItem } from './types';
+import { UserId } from '../user/types';
+import { ProductId, Price } from '../product/types';
+import { NotFoundException } from '@nestjs/common';
+import { Order as DomainOrder } from './types'; // Domain Order interface from types.ts
```

ESLint

[error] 'DomainOrder' is defined but never used.

(@typescript-eslint/no-unused-vars)